js/views/modals/connectionManagement/StatusBar.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'underscore';
+import { capitalize } from '../../../utils/string';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super(options);
+
+    this._state = {
+      msg: '',
+      ...options.initialState || {},
+    };
+  }
+
+  className() {
+    return 'statusBarMessage ';
+  }
+
+  events() {
+    return {
+      'click .js-btnClose': 'onClickClose',
+      'click .statusMsg [class^="js-"], .statusMsg [class*=" js-"]': 'onMsgContentClick',
+    };
+  }
+
+  onClickClose() {
+    this.trigger('closeClick');
+  }
+
+  // User of this view may embed CTA's in the msg. This is a generic handler to
+  // trigger events for them.
+  onMsgContentClick(e) {
+    // If the the el has a '.js-<class>' class, we'll trigger a
+    // 'click<Class>' event from this view.
+    const events = [];
+
+    e.currentTarget.classList.forEach((className) => {
+      if (className.startsWith('js-')) events.push(className.slice(3));
+    });
+
+    if (events.length) {
+      events.forEach(event => {
+        this.trigger(`click${capitalize(event)}`, { view: this, e });
+      });
+    }
```

Comment:
Hmm... maybe, but I often am scoping it to some selector, which I suppose could be passed in as an option. I'm not sure this is global enough to put in the BaseView, so maybe I'd put this in a new view that extends from the BaseView.

It's a good thought. I'll keep it in mind and if I find I keep re-using it, I'll think of the best way to centralize it.